packages/features/ee/workflows/lib/constants.ts

```diff
@@ -14,11 +14,33 @@ export const WORKFLOW_ACTIONS = [
   WorkflowActions.EMAIL_ADDRESS,
   WorkflowActions.SMS_ATTENDEE,
   WorkflowActions.SMS_NUMBER,
+  WorkflowActions.WHATSAPP_ATTENDEE,
+  WorkflowActions.WHATSAPP_NUMBER,
 ] as const;
 
 export const TIME_UNIT = [TimeUnit.DAY, TimeUnit.HOUR, TimeUnit.MINUTE] as const;
 
-export const WORKFLOW_TEMPLATES = [WorkflowTemplates.CUSTOM, WorkflowTemplates.REMINDER] as const;
+export const WORKFLOW_TEMPLATES = [
+  WorkflowTemplates.CUSTOM,
+  WorkflowTemplates.REMINDER,
+  WorkflowTemplates.REMINDER,
```

Comment:
```suggestion
```

Reply:
I've reverted the changes to remove the "CANCELLED" and "RESCHEDULED" templates

Reply:
Can you remove the duplicate `WorkflowTemplates.REMINDER` in `WORKFLOW_TEMPLATES` as well please

Reply:
I just did